todo_app/app.py

```diff
@@ -1,11 +1,32 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item, get_item, save_item, remove_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    items = sorted(items, key = lambda item: 1 if item["status"] == "Completed" else 0 )
+    return render_template("index.html", items=items)
+
+@app.route('/add-todo', methods=['POST'])
+def add_todo():
+    add_item(request.form.get("title"))
+    return redirect("/")
+
+@app.route('/remove-todo/<id>', methods=['POST'])
+def remove_todo(id):
+    remove_item(id)
+    return redirect("/")
+
+#I know this is the incorrect use of post but to use put I need to use JQuery for the form submission
```

Comment:
Almost true - you would need JavaScript to use other HTTP methods but not necessarily jQuery which is just a JavaScript library.

But yes we expect you to just use POST for all these endpoints rather than introduce another language. Semantic HTTP methods are more important when building a RESTful API.